assets/styles/styles.css

```diff
@@ -47,7 +47,7 @@
 .conclude {
   font-style: italic;
```

Comment:
you can also put a paragraph in italic in the html file by using the tag <i></i>